pkg/base1/cockpit.js

```diff
@@ -2808,7 +2826,6 @@ function full_scope(cockpit, $, po) {
                     var message_len = message.length;
                     if (message_len === undefined) {
                         meta = message;
-                        last = null;
```

Comment:
It is up to the source whether compression is done or not across a meta message.  If it is not possible, then it will simply not be done.